ts/Core/Defaults.ts

```diff
@@ -2873,6 +2905,13 @@ function setOptions(
         });
     }
 
+    if (options.lang?.chartTitle) {
```

Comment:
Kan we skip this condition and just do `text: options.lang?.chartTitle`. Because if it's undefined then there's no harm done.

Reply:
Not quite with the current setup it seems, as undefined is disabled. We could do this, but I suspect the `if` is cheaper:

```ts
merge(true, defaultOptions.title, {
     text: options.lang?.chartTitle ?? defaultOptions.title?.text
});
```